package.json

```diff
@@ -55,7 +54,7 @@
         "react-json-tree": "^0.11.2",
         "sass-loader": "^8.0.0",
         "style-loader": "^1.0.0",
-        "three": "^0.117.1",
+        "three": "^0.119.0",
```

Comment:
119.1 ?

Reply:
I checked the peerDependencies property in the package.json, 0.119.0 is correct